lib/SIL/SILVerifier.cpp

```diff
@@ -56,6 +56,10 @@ static llvm::cl::opt<bool> AbortOnFailure(
                               "verify-abort-on-failure",
                               llvm::cl::init(true));
 
+static llvm::cl::opt<bool> VerifyDIHoles(
```

Comment:
Is this option really necessary, shouldn't it just always be on?

Reply:
I think this is good while we're transitioning, so that if something breaks, we can just turn the option off and investigate rather than reverting the whole commit.